crates/fs/public/directory.rs

```diff
@@ -425,16 +429,17 @@ impl PublicDirectory {
     /// }
     /// ```
     pub async fn store<B: BlockStore>(&self, store: &mut B) -> Result<Cid> {
-        let bytes = self.encode(store).await?;
-        store.put_block(bytes, IpldCodec::DagCbor).await
+        store.put_async_serializable(self).await
+        // self.flush(store).await?;
+        // store.put_serializable(self).await
```

Comment:
I know this code is probably WIP. I'm making sure we don't forget to remove the commented lines :sweat_smile: